spec/models/adviser_spec.rb

```diff
@@ -29,6 +29,26 @@
     end
 
     describe 'geographical coverage' do
+      describe 'does or does not cover whole of uk' do
+        it 'is required' do
+          expect(build(:adviser, covers_whole_of_uk: nil)).to_not be_valid
+        end
+
+        context 'when not selected' do
```

Comment:
This is essentially the spec to ensure that the "fix" for #5659 doesn't regress - I'd prefer to not have this.